src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -123,13 +141,22 @@ function presentTransaction (transaction) {
 
 const HoloFuelDnaInterface = {
   user: {
-    get: async ({ agentId }) => {
-      const result = await createZomeCall('transactions/whoami')({ agentId })
-      if (result.error) throw new Error('There was an error locating the agent nickname. ERROR: ', result.error)
+    get: async () => {
+      const result = await createZomeCall('transactions/whoami')()
+      if (result.error) throw new Error('There was an error locating the current holofuel agent nickname. ERROR: ', result.error)
       return {
         id: result.pub_sign_key,
         nickname: result.nick
       }
+    },
+    getCounterparty: async ({ agentId }) => {
+      const result = await createZomeCall('transactions/whoami')({ agentId })
+      if (result.error) throw new Error('There was an error locating the counterparty agent nickname. ERROR: ', result.error)
+
+      return {
+        pubkey: result.pub_sign_key,
```

Comment:
This should return 
```
{
  id: agentId,
  pubkey: result.pub_sign_key,
  nickname: result.nick
}
```
So that the returned `HolofuelUser` type has an id.